test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -15,6 +16,30 @@ if (dialect.match(/^postgres/)) {
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+    describe('createTable', function () {
+      beforeEach(function () {
+        var self = this;
+        return this.queryInterface.dropTable('Group').then(function () {
+          return self.queryInterface.createTable('Group', {
+            enum1: {type: DataTypes.ENUM, values: ['1', '2']}
+          });
+        });
+      });
+
+      it('should only create enum types for columns in creates', function () {
+        var self = this;
+        return self.queryInterface.createTable('Group', {
```

Comment:
Why are you calling createTable  twice  - isn't the original issue about addColumn?